src/blocks/Header/MobileHeader.js

```diff
@@ -68,11 +69,49 @@ const MobileHeader = ({ headerProps }) => {
     animated: true,
     unstable_initialFocusRef: null,
   })
+
+  const [sequence, setSequence] = useState({
+    segments: [0, 1],
+    forceFlag: true,
+  })
+
+  const animate = () => {
+    console.log('run')
+    console.log(sequence)
+    let start = sequence.segments[0]
+    let stop = sequence.segments[1]
+
+    if (stop === 1 || stop === 90) {
+      start = 0
+      stop = 50
+    } else {
+      start = 50
+      stop = 75
+    }
+
+    setSequence({
+      segments: [start, stop],
+      forceFlag: true,
+    })
+  }
+
```

Comment:
I don't think that this is necessary. Can you please check if everything works without it? 🙏

Reply:
Sorry, I forgot to remove segment function which I created. Fixing it.

Reply:
Fixed it